components/FeatureHighlight/examples/FeatureHighlightColorExample.m

```diff
@@ -15,6 +15,7 @@
  */
 
 #import "MaterialFeatureHighlight.h"
+#import "MDCFeatureHighlightAccessibilityMutator.h"
```

Comment:
Naming/nit: Prefer `MDCFeatureHighlightTextAccessibilityMutator.h` since you aren't mutating the outerColor or innerColor.

Reply:
I think it's better to leave the class name as is, we could add more functionality to it if needed.